### PR TITLE
Adds `get_config` command.

### PR DESCRIPTION
**Pull request by ser-io**
- The new command is an utility to fetch the final configuration parameter values.

### CODE REVIEW

File: pkg/cli/cli.go [hunk 1]
```diff
@@ -1064,6 +1073,24 @@ func runDisconnectCommand(flags *ConnectFlags, c *command, args []string, opts *
 	return merr
 }
 
+func runGetConfigCommand(c *cobra.Command, args []string, cfg Config) error {
+	if len(args) == 0 {
+		return errors.New("missing parameter")
```
Comment (jemoreira) [inline pkg/cli/cli.go:1078]: nit: missing config property name

File: pkg/cli/cli.go [hunk 2]
```diff
@@ -1064,6 +1073,24 @@ func runDisconnectCommand(flags *ConnectFlags, c *command, args []string, opts *
...
+	}
+	if len(args) > 1 {
+		return errors.New("use one parameter at a time")
```
Comment (jemoreira) [inline pkg/cli/cli.go:1081]: nit: same

File: pkg/cli/cli.go [hunk 3]
```diff
@@ -1064,6 +1073,24 @@ func runDisconnectCommand(flags *ConnectFlags, c *command, args []string, opts *
...
+	}
+	switch args[0] {
```
Comment (jemoreira) [inline pkg/cli/cli.go:1083]: nit: You could implement this once and never have come back to it again by encoding the config object as JSON and then decoding it using a `map[string]interface{}`.
You'd have to split the "property name" by `'.'` to support inner fields, but that shouldn't be too hard.
We are already planning to support multiple service urls, which means the config structure could change, so don't reply with "YAGNI" :)

**Review by jemoreira — APPROVED** [review]

File: pkg/cli/cli.go (hunk 1 above)
Comment (ser-io) [reply pkg/cli/cli.go]: done

File: pkg/cli/cli.go (hunk 2 above)
Comment (ser-io) [reply pkg/cli/cli.go]: done

File: pkg/cli/cli.go (hunk 3 above)
Comment (ser-io) [reply pkg/cli/cli.go]: :) done.

File: pkg/cli/config.go [hunk 4]
```diff
@@ -27,22 +27,22 @@ import (
...
 
 type Config struct {
-	ServiceURL                string
-	Zone                      string
-	HTTPProxy                 string
-	ConnectionControlDir      string
-	KeepLogFilesDays          int
-	BuildAPICredentialsSource string
-	Host                      HostConfig
+	ServiceURL                string     `json:"service_url,omitempty"`
```
Comment (jemoreira) [inline pkg/cli/config.go:39]: Doesn't this break parsing the config files?
The config file has `ServiceURL` instead of `service_url`

Comment (ser-io) [reply pkg/cli/config.go:39]: it does not break, the toml parsing do not uses the json annotations.

**Pull request merged**